kustomize/pgadmin/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: postgres-operator
+
+resources:
+- pgadmin.yaml
```

Comment:
⛏️ Can you add a newline at the end of this?

Reply:
I think I did, can you double check? I also added the readme content

Reply:
Thx!